mlflow/tensorflow.py

```diff
@@ -229,22 +242,34 @@ def load_model(model_uri, tf_sess):
     >>>     output_tensors = [tf_graph.get_tensor_by_name(output_signature.name)
     >>>                       for _, output_signature in signature_def.outputs.items()]
     """
+
+    if LooseVersion(tensorflow.__version__) < LooseVersion('2.0.0'):
+        if not tf_sess:
+            tf_sess = tensorflow.get_default_session()
+            if not tf_sess:
+                raise ValueError("No active session found while trying to use load_model().")
```

Comment:
```suggestion
                raise ValueError("No TensorFlow session found while calling load_model(). You can set the default Tensorflow session before calling load_model via `session.as_default()`, or directly pass a session in which to load the model via the tf_sess argument.")
```

Basically would be good to give the user an actionable error message telling them how to fix the issue (this is in general a good thing to do).